src/content/docs/dropins/checkout/tutorials/add-payment-method.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Add a payment method
+description: Learn how to integrate third-party payment providers with the Commerce boilerplate template.
+---
+
+import Aside from '@components/Aside.astro';
+import FileTree from '@components/FileTree.astro';
+import { Tabs, TabItem, Code, Steps } from '@astrojs/starlight/components';
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+
+To integrate third-party payment providers, you can use the extensibility features provided by the checkout drop-in component. This component allows you to customize the list of payment methods shown during the checkout process using slots.
+
+## Steps-by-step
+
+This tutorial provides a step-by-step guide for integrating the Braintree payment provider with the Commerce boilerplate template. You can use these steps as a reference to integrate other payment providers as well.
+
+<Aside type="note">
+Make sure that the payment provider you choose offers an SDK or API that you can use to integrate with the [slots](/dropins/checkout/slots/) provided by the `PaymentMethods` container in the checkout drop-in component.
```

Comment:
@tkacheva, is there anything else that we can/should explain about using payment providers other than Braintree?

Reply:
@keoko & @mcocapiza,

If a customer chooses a different payment provider, I assume it must be a provider that 1) Adobe Commerce supports and 2) provides an extension that must be installed/configured on the Adobe Commerce backend (unless already bundled with core Commerce like Braintree).

Can you please confirm if that's true so we can clarify in the public docs?

Reply:
Thanks @sirugh. I mean "merchant".